Mutagen.Bethesda.Analyzers.SDK/Analyzers/IsolatedRecordFrameAnalyzerParams.cs

```diff
@@ -34,8 +39,20 @@ public void AddTopic(
         IFormattedTopicDefinition formattedTopicDefinition,
         params (string Name, object Value)[] metaData)
     {
+        string? editorId = null;
+        if (Frame.TryFindSubrecord(RecordTypes.EDID, out var edid))
+        {
+            editorId = edid.AsString(GameConstants.Get(_mod.GameRelease).Encodings.NonLocalized);
+        }
```

Comment:
I think we should move this logic upstream to whoever is driving.   The RecordFrameDriver making IsolatedRecordFrameAnalyzerParams knows the mod, the frame, and all that.  It can do the EDID extraction and MajorRecordIdentifier construction once ahead of time and just pass it in?    Feels more appropriate, rather than doing the work for each AddTopic